openhands_cli/mcp/mcp_display_utils.py

```diff
@@ -0,0 +1,151 @@
+"""Utility functions for displaying MCP server information.
+
+This module provides shared utilities for extracting and formatting MCP server
+configuration details across different display contexts (CLI, TUI, etc.).
+"""
+
+from typing import Any
+
+
+def extract_server_info(server_spec: dict[str, Any]) -> dict[str, Any]:
```

Comment:
NIT: i'd prefer some sort of pydantic class for the "Info" here but it is probably also not super necessary since it only got used once here

And we already have configuration like `MCPConfig` from `fastmcp`